src/components/Behavior/Behavior.tsx

```diff
@@ -0,0 +1,25 @@
+import { Checkbox, Flex, Text } from '@chakra-ui/react'
+
+type BehaviorProps = {
+  behaviorName: string;
+  behaviorDescription: string;
+  youtubeEmbed: any[];
+};
+
+const Behavior = ({ behaviorName, behaviorDescription, youtubeEmbed }:BehaviorProps) => {
+  return (
+    <Flex direction="column">
+      <Checkbox as="b">
+        { behaviorName }
+      </Checkbox>
+      <Text>
+        { youtubeEmbed }
```

Comment:
since text is a <p> and youtubeEmbed has a div,
We are receiving
<p>
<div></div>
</p>

this structure is not appropriate since div cannot be a descendant of p! I have made a change to adjust for this.